cloudbuild.yaml

```diff
@@ -0,0 +1,43 @@
+steps:
+  - name: node:12-slim
+    id: install
+    entrypoint: yarn
+    args: ["install"]
+    waitFor: ["-"]
+
+  - name: node:12-slim
+    entrypoint: yarn
+    args: ["test"]
+    waitFor: ["install"]
+
+  - name: "gcr.io/kaniko-project/executor:latest"
+    args:
+      [
+        "--destination=gcr.io/$PROJECT_ID/kibana-bot:$COMMIT_SHA",
+        "--cache=true",
+      ]
+    waitFor: ["-"]
+
+  - name: "gcr.io/cloud-builders/gcloud"
+    args:
+      [
+        "beta",
+        "run",
+        "deploy",
+        "kibana-bot",
+        "--image",
+        "gcr.io/$PROJECT_ID/kibana-bot:$COMMIT_SHA",
+        "--region",
+        "us-central1",
+        "--platform",
+        "managed",
+        "--allow-unauthenticated",
+        "--concurrency",
+        "80",
```

Comment:
Is this the max concurrency of the container? What happens when more than 80 concurrent requests are received?

Reply:
It's supposed to be the max number of requests a container can service at a time.

There's not a clear answer in the docs, so I did a little testing... It looks like it rejects requests (it doesn't queue them up or similar) with HTTP 429 once the maximum is hit. 80 is the maximum value. Beyond that, you're supposed to allow Cloud Run to scale up the number of instances.